x/fees/keeper/msg_server.go

```diff
@@ -13,6 +13,11 @@ import (
 
 var _ types.MsgServer = &Keeper{}
 
+const (
+	// keccak256(word) costs 36 gas
+	addressDerivationCostCreate uint64 = 50
+)
```

Comment:
IMO we should add a `AddrDerivationCostCreate` parameter so that it can be configured via governance 👍

Reply:
Done.